PokeAlarm/Events/QuestEvent.py

```diff
@@ -0,0 +1,66 @@
+# Standard Library Imports
+import datetime
+# 3rd Party Imports
+# Local Imports
+from PokeAlarm import Unknown
+from . import BaseEvent
+from PokeAlarm.Utils import get_gmaps_link, get_applemaps_link, \
+    get_dist_as_str
+
+
+class QuestEvent(BaseEvent):
+    """ Event representing the discovery of a PokeStop. """
+
+    def __init__(self, data):
+        """ Creates a new Quest Event based on the given dict. """
+        super(QuestEvent, self).__init__('quest')
+
+        # Identification
+        self.stop_id = data['pokestop_id']
+        self.stop_name = data['name']
```

Comment:
It seems a bit ambiguous to have `name` come from the webhook when `name` can also be interpreted as `quest`.
I suggest renaming to `pokestop_name` to match `pokestop_id`.